### ISSUE TITLE
~Media/Press Page~

### ISSUE THREAD

**Issue by urentmellc** [pixelphpdev/urentme#77]
Please see examples fo you to create a page for us...

https://www.thumbtack.com/press/
https://porch.com/about/press#newsroom

**Comment by Jayantapixel** [comment]
Designer is review both and will work and show you tomorrow.

**Comment by urentmellc** [comment]
ok thank you.

**Comment by urentmellc** [comment]
Again, nothing. Where is that page??? Send it now.

**Comment by Jayantapixel** [comment]
Please find below media pages (both listing and details)

![media](https://user-images.githubusercontent.com/35031627/40782846-38b59b92-64fe-11e8-9bf7-faf97cf3d195.png)

![media 2](https://user-images.githubusercontent.com/35031627/40782843-384fac88-64fe-11e8-9533-a73820baaa68.png)

**Comment by urentmellc** [comment]
I like it think it will work... Thank you. I will need to be able to edit all content/video on back end... Correct?

**Comment by pixelphpdev** [comment]
Yes. We are working on this page.We will send this page link after complete.

**Comment by urentmellc** [comment]
Push the add a listing to any user live, what is the problem? 
		
		

		Warm Regards,
Anthony HorovitzChief Operating Officer, Founder, URentMe.Com855-400-0082 | 702-219-0186 | anthony@urentme.com | www.urentme.com | Skype: anthony.horovitz | 871 Coronado Center Drive, Suite 200, Henderson, NV 89052
	




On Fri, Jun 1, 2018 at 8:54 AM -0600, "pixelphpdev" <notifications@github.com> wrote:












Yes. We are working on this page.We will send this page link after complete.

**Comment by urentmellc** [comment]
where are we at? Need this my morning today done. Not waiting anymore.

**Comment by Jayantapixel** [comment]
Please send few content for media page. Its almost done but need some changes after getting the actual content. You can have a look but its not the final version.
https://urentme.com/dev/pages/media

We will release it after getting few content from you tomorrow our afternoon. Also let me know, from where this page will be linked? I feel we can add one item as Media in footer.

**Comment by urentmellc** [comment]
This doesnt look good the way it was shown before. You will need to rework the design/page. I will work on getting some content to share on the page. Please confirm. Thank you.

**Comment by Jayantapixel** [comment]
Listing page is same as design but details page will be change specially on image/video are when we got some actual content.

**Comment by urentmellc** [comment]
Ok will add some content for you shortly to use here.

**Comment by urentmellc** [comment]
Here is something live to add... 

http://rvdailyreport.com/owner/new-urentme-offers-earning-opportunities-for-rv-owners/

**Comment by urentmellc** [comment]
here is something for you to use on our media page....

http://rvdailyreport.com/owner/new-urentme-offers-earning-opportunities-for-rv-owners/

**Comment by Jayantapixel** [comment]
Media page done in Dev. Please check same link.
https://urentme.com/dev/pages/media

Also let us know from where we need to link this page. Do we put a media link in the footer?

**Comment by pixelphpdev** [comment]
We have lived media page. But Let us know, where we add media page link?
